Classes/Dto/Product.php

```diff
@@ -255,20 +290,49 @@ public function addImage(PersistentResource $persistentResource, string $type):
         return $this;
     }
 
+    /**
+     * @return PersistentResource[]
+     */
+    public function getUploadResources(): array
+    {
+        return $this->uploadResources;
+    }
+
     /**
      * @return string[]
      */
     public function getImages(): array
     {
-        return $this->imageTypes;
+        return $this->images;
     }
 
     /**
-     * @return PersistentResource[]
+     * @param string $type
+     * @return string
      */
-    public function getUploadResources(): array
+    public function getImagePathByType(string $type = 'test'): string
```

Comment:
wouldn´t `$type = 'main'`make more sense here?

Reply:
ups, yes you are right